layouts/partials/header.html

```diff
@@ -1,7 +1,7 @@
 <div id="cellophane" onclick="kub.toggleMenu()"></div>
 
 <header>
-    <a href="{{ site.Home.RelPermalink }}" class="logo" title="Production-Grade Container Orchestration - Kubernetes" aria-label="Kubernetes website"></a>
+    <a href="{{ site.Home.RelPermalink }}" class="logo" title="{{ site.Home.Title }} - {{ site.Params.title }}" aria-label="Kubernetes website"></a>
```

Comment:
@sftim, changes seem okay. Is there any value in using { site.Params.description } instead of {{ site.Home.Title }}?

Reply:
I don't know Hugo well enough to tell which is better. You recommend `title="{{ site.Params.description }} - {{ site.Params.title }}"`?

Reply:
I don't which variable is best to set the localized version of the title.
`site.Home` is used in `top-menu.html` (layouts/partials/docs/top-menu.html).
I noticed that a number of the localization teams have translated the `description` string in `config.toml`.

Reply:
It makes sense to use the site variable as long as the localization teams know to translate the `title` field in their home page.